Makefile

```diff
@@ -20,6 +20,8 @@ refresh-workspace-hosts-dev:
 
 dev: start-support
 	@yarn dev
+dev-bun:
+	@yarn dev-bun
```

Comment:
Bun is very quick at restarting in dev mode, usually <2s, which is much faster than nodemon + Node. Fast iteration was really nice while developing this, as there was a lot of custom CSS to fine-tune. I figured I might as well add this Makefile target for others who want to take advantage of that.

Reply:
Nice! Works for me 🚀